kombu/transport/redis_cluster.py

```diff
@@ -320,8 +324,7 @@ def _brpop_read(self, **options):
 
             conn.client.connection.send_command('BRPOP', conn.key, 1) # schedule next BRPOP
```

Comment:
the timeout value should be passed from caller like `brpop_start`, I believe. How about move this `send_command` to `_brpop_start`?